test/verify/check-multi-machine

```diff
@@ -504,6 +504,7 @@ class TestMultiMachine(MachineCase):
         b.enter_page("/dashboard")
         # Check the server image
         b.wait_present("#dashboard-hosts a[data-address='%s']" % m2.address)
+        b.wait_present("#dashboard-hosts a[data-address='%s'] img" % m2.address)
```

Comment:
If ```#dashboard-hosts a[data-address='%s']``` is not present then ```#dashboard-hosts a[data-address='%s'] img``` can error, instead of waiting. Having both makes sure it waits for the elements to appear properly.